backend/src/main/java/com/praksa/auction/dto/BasicInfoDto.java

```diff
@@ -0,0 +1,50 @@
+package com.praksa.auction.dto;
+
+public class BasicInfoDto {
+    private long id;
+    private String firstName;
+    private String lastName;
+    private String email;
+
+    public BasicInfoDto() {
```

Comment:
BasicInfo ?

Reply:
This is basic information about user that will be returned to frontend after login. It was created in order to reduce data about user that is sent from backend to frontend.

Reply:
Can we add some better naming for this class, it could be basic info for a lot of things,
BasicUserInfoDto maybe ?